Migrate the old code from the original repo

SilasKenneth opened the pull request:
### Changes made to the original repo.
- Changing all the references of the original repo to the beta repo.
- Adding the static analysis test to the pipeline.
- Deleted the V1 models folder from the src folder.
- Deleted core components that will be provided by the core package.
- Updated the tests to use the beta models instead of the v1 models.

MIchaelMainer commented:
GitHub appears to try to load the first 3000 files into the browser killing the review so I'll provide review here.

1. Can we leave Enum.php in core since it is shared by both v1.0 and beta?
2. Comment only - PHP 8.1 will have proper enumerations built in. https://wiki.php.net/rfc/enumerations
3. Let's put all files in this repo in the beta namespace. `Beta\Microsoft\Graph\`
4. I don't think we need to have a Core directory. Move those files up (if we keep them).
5. GraphConstants - update namespace if we keep this. I think we only want to keep SDK_VERSION and maybe API_VERSION.

If you want to work on these in a separate PR, I'm good with that. Create issues for the, This PR is not reviewable in GH due to its size.

SilasKenneth commented:
> GitHub appears to try to load the first 3000 files into the browser killing the review so I'll provide review here.
> 
>     1. Can we leave Enum.php in core since it is shared by both v1.0 and beta?
> 
>     2. Comment only - PHP 8.1 will have proper enumerations built in. [wiki.php.net/rfc/enumerations](https://wiki.php.net/rfc/enumerations)
> 
>     3. Let's put all files in this repo in the beta namespace. `Beta\Microsoft\Graph\`
> 
>     4. I don't think we need to have a Core directory. Move those files up (if we keep them).
> 
>     5. GraphConstants - update namespace if we keep this. I think we only want to keep SDK_VERSION and maybe API_VERSION.
> 
> 
> If you want to work on these in a separate PR, I'm good with that. Create issues for the, This PR is not reviewable in GH due to its size.
> 
> Requesting feedback.

I have made the following changes from the feedback.
- Removed `Enum.php` file.
- I have moved the directories inside `src/Beta/Microsoft/Graph` to `src` and changed the namespace in the `composer.json`.
- Removed all the constants inside the `GraphConstants.php` file and moved the file to the `src` folder. I have kept the file(`GraphConstants.php`) since it is needed by tag creation action and the script to increment the version number.

MIchaelMainer reviewed (approved):
Looks good
